hushline/routes.py

```diff
@@ -232,8 +228,8 @@ def register() -> Response | str:
                 flash("💔 Username already taken.", "error")
                 return redirect(url_for("register"))
 
-            password_hash = bcrypt.generate_password_hash(password).decode("utf-8")
-            new_user = User(primary_username=username, password_hash=password_hash)
+            new_user = User(primary_username=username)
```

Comment:
From a previous thread, instead of setting this separately, override the `__init__` method and let `password_hash` be a keyword arg

Reply:
Got it.